GettingStarted.md

```diff
@@ -0,0 +1,114 @@
+
+This document summarises the proces of building your first package on 
+OBS.  It only skims the surface of OBS and is not a manual.
+
+To start building your packages on OBS you will require an account.
+Follow the process in the README and contact 'lbt' for an account to 
+be set up.
+
+When your account is set up, you will have a "Home Project".  This is where
+you can start creating packages.  The URL for your home project will be:
+https://build.sailfishos.org/project/show/home:<username>
+
+## Creating a package on OBS
+
+Projects contain packages which are built against target repositories
+(sailfish versions).  To create your first package, perform the following
+steps:
+
+1. Click "Create Package"
+2. On the new package page, give the package a name and click "Save Changes"
+   * eg harbour-mypackage
+3. You now need to add files to the package to allow OBS to get the source and
+build it.  Create a file on your computer called `_service` with the following content:
+```
+<services>
+    <service name="tar_git">
+    <param name="url">URL TO PROJECT</param>
+    <param name="branch">master</param>
+    <param name="revision"></param>
+    </service>
+</services>`
+```
+   And specify at least the URL.  If you want to build another branch, change the 
+   branch name, and if you want to build a particular revision, specify a git hash
+   or tag in the revision.
+4. Back on OBS, click "Add File" then "Browse" and select the `_service` file you
+   just created and "Save Changes"
+5. If all goes well, OBS will fetch the source code.
+
+## Adding target repositories
+
+Back at the home project on OBS, you will require target repositories to build against.
+These can be added using a UI but it is easier to add them manually.
+
+1. Click "Advanced" and "Meta"<br>
+   Note, dont confuse this with the package Meta configuration, the nanes are the same
+2. You will be presented with an XML description of the target repositories.  Between
+   the ```<project> </project>``` tags, add the following:
+```
+<repository name="sailfish_latest_i486">
+    <path project="sailfishos:latest" repository="latest_i486"/>
+    <arch>i586</arch>
+</repository>
+<repository name="sailfish_latest_armv7hl">
+    <path project="sailfishos:latest" repository="latest_armv7hl"/>
+    <arch>armv8el</arch>
+</repository>
+<repository name="sailfish_latest_aarch64">
+    <path project="sailfishos:latest" repository="latest_aarch64"/>
+    <arch>aarch64</arch>
+</repository>
+<repository name="sailfish_4.1.0.24_armv7hl">
+    <path project="sailfishos:4.1.0.24" repository="latest_armv7hl"/>
+    <arch>armv8el</arch>
+</repository>
+<repository name="sailfish_4.1.0.24_aarch64">
+    <path project="sailfishos:4.1.0.24" repository="latest_aarch64"/>
+    <arch>aarch64</arch>
+</repository>
+<repository name="sailfish_4.1.0.24_i486">
+    <path project="sailfishos:4.1.0.24" repository="latest_i408"/>
+    <arch>i586</arch>
+</repository>
+```
```

Comment:
The proceeding list items get messed up because of the code block. It seems [the solution](https://gist.github.com/clintel/1155906) is to indent the fenced code block (suggestion untested).

````suggestion
    ```
    <repository name="sailfish_latest_i486">
        <path project="sailfishos:latest" repository="latest_i486"/>
        <arch>i586</arch>
    </repository>
    <repository name="sailfish_latest_armv7hl">
        <path project="sailfishos:latest" repository="latest_armv7hl"/>
        <arch>armv8el</arch>
    </repository>
    <repository name="sailfish_latest_aarch64">
        <path project="sailfishos:latest" repository="latest_aarch64"/>
        <arch>aarch64</arch>
    </repository>
    <repository name="sailfish_4.1.0.24_armv7hl">
        <path project="sailfishos:4.1.0.24" repository="latest_armv7hl"/>
        <arch>armv8el</arch>
    </repository>
    <repository name="sailfish_4.1.0.24_aarch64">
        <path project="sailfishos:4.1.0.24" repository="latest_aarch64"/>
        <arch>aarch64</arch>
    </repository>
    <repository name="sailfish_4.1.0.24_i486">
        <path project="sailfishos:4.1.0.24" repository="latest_i408"/>
        <arch>i586</arch>
    </repository>
    ```
````